src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/Pkcs8PrivateKeyInfo.cs

```diff
@@ -197,8 +197,7 @@ public bool TryEncode(Span<byte> destination, out int bytesWritten)
             }
             finally
             {
-                CryptographicOperations.ZeroMemory(decryptedMemory.Span);
-                CryptoPool.Return(decrypted.Array!, clearSize: 0);
+                CryptoPool.Return(decrypted);
```

Comment:
I'll stop commenting on these 0 to all changes... I'm getting the sense they were intentional :)